src/components/v2/appDetails/sourceInfo/scaleWorkloads/ScaleWorkloadsModal.component.tsx

```diff
@@ -80,6 +89,24 @@ export default function ScaleWorkloadsModal({ appId, onClose, history }: ScaleWo
         }
     }, [appDetails])
 
+    const _getAndSetAppDetail = async () => {
+        if (appDetails?.deploymentAppType === DeploymentAppType.argo_cd) {
```

Comment:
We can write this condition inside try so that no need to setdetails in else part